test/performance/kshelper.py

```diff
@@ -51,48 +50,58 @@ def load_kubeconfig(self, kubeconfig, context):
         
         api_client = kubernetes.client.ApiClient()
         core_v1_api = kubernetes.client.CoreV1Api(api_client)
+        app_v1_api  = kubernetes.client.AppsV1Api(api_client)
         custom_objects_api = kubernetes.client.CustomObjectsApi(api_client)
 
         os.remove(tmp_config)
-        return core_v1_api, custom_objects_api
+        return core_v1_api, app_v1_api, custom_objects_api
 
-    def get_crd_time(self, grp, v, pl, name=None, check=False):
+    def get_custom_objects_time(self, fname, grp, v, pl, name=None, ns=None, bindingPolicy=None, check=False, single=True):
         try:
-            if name != None and check == False:
-                api_response = self.custom_objects_api.list_cluster_custom_object(group=grp, version=v, plural=pl, field_selector='metadata.name=' + name)
+            if name == None and ns != None and bindingPolicy != None:
+               api_response = self.custom_objects_api.list_namespaced_custom_object(group=grp, version=v, plural=pl, namespace=ns, label_selector='transport.kubestellar.io/originOwnerReferenceBindingKey=' + bindingPolicy)
+            elif  name != None and ns != None and bindingPolicy != None:
+               api_response = self.custom_objects_api.list_namespaced_custom_object(group=grp, version=v, plural=pl, namespace=ns, label_selector='transport.kubestellar.io/originOwnerReferenceBindingKey=' + bindingPolicy)
+            elif name != None and ns != None and bindingPolicy == None:
+               api_response = self.custom_objects_api.list_namespaced_custom_object(group=grp, version=v, plural=pl, namespace=ns, field_selector='metadata.name=' + name)
             else:
-                api_response = self.custom_objects_api.list_cluster_custom_object(group=grp, version=v, plural=pl)
+               api_response = self.custom_objects_api.list_cluster_custom_object(group=grp, version=v, plural=pl)
+
+            if name == None or single == False:
+               # Create the output file
+               if os.path.exists(fname):
+                  mode = 'a' # append if already exists
+               else:
+                  mode = 'w' # make a new file if not
```

Comment:
Doesn't mode 'a' do the right thing in both cases?

```shell
mspreitz@mjs13 kubestellar % python3
Python 3.12.5 (main, Aug  6 2024, 19:08:49) [Clang 15.0.0 (clang-1500.3.9.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> fp = open('fubar', 'a')
>>> fp
<_io.TextIOWrapper name='fubar' mode='a' encoding='UTF-8'>
>>> fp.write('xyz')
3
>>> fp.close()
>>> ^D

mspreitz@mjs13 kubestellar % ls -l fubar
-rw-r--r--  1 mspreitz  staff  3 Sep  6 14:02 fubar
```

Reply:
You're correct! I did similar test and I got the same results. Fixed!